packages/jaeger-ui/.depcheckrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ignores": [
+    "@babel/preset-env",
+    "@babel/preset-react",
+    "@babel/preset-typescript",
+    "@types/react-dom",
+    "@types/react-router-redux",
```

Comment:
these are just type declarations, why are they needed if no code imports them?

Reply:
Type declarations are never imported inside the codebase. They are supplemented so that typescript doesn't throw type errors for the custom functions or components that are used inside the project.

Other type declarations are also not imported anywhere but are essential because of typescript compatibility.

`@types/react-dom` is for `react` and `@types/react-router-redux` is for `react-router-redux` and both of these packages are being used inside the project, so having their type declarations are necessary IMO.

Reply:
I couldn't find an exact tool for that, but there's another tool `npm-check` that uses `depcheck` behind the scenes.

I checked the results of `npm-check` and it didn't show any of `@types/xxx` dep as unused. Though, all other results were same. Maybe its parser is more correct.